SearchBayesianSkopt: Config 0 Exception. Config: {'num_factors': 392, 'regularization': 8.838066162587926e-05, 'learning_rate': 0.06018800534643989, 'epochs': 300} - Exception: Traceback (most recent call last):
  File "/Users/tangtang.zhou/Documents/GitHubProjects/recsys_2019/course_lib/ParameterTuning/SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "/Users/tangtang.zhou/Documents/GitHubProjects/recsys_2019/course_lib/ParameterTuning/SearchAbstractClass.py", line 237, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_parameters)
  File "/Users/tangtang.zhou/Documents/GitHubProjects/recsys_2019/course_lib/ParameterTuning/SearchAbstractClass.py", line 229, in _fit_model
    **current_fit_parameters)
  File "/Users/tangtang.zhou/Documents/GitHubProjects/recsys_2019/src/model/MatrixFactorization/FunkSVDRecommender.py", line 32, in fit
    self.model.predict()
TypeError: predict() missing 2 required positional arguments: 'uid' and 'iid'

